### DISCUSSION TITLE
Grist tables integration

### DISCUSSION THREAD

**Discussion by helmut72** [outline/outline#3851] [Ideas]
Would be cool to integrate tables from Grist, self hosted and cloud version.

https://www.getgrist.com

This would be a self hosted alternative to Airtable for the ones, who als self host Outline.

I have replaced the Airtable fqdn in airtable.js with my self hosted Grist installation, adjusted the regex, restarted Outline, but Airtable integration still accept Airtable links only.

Thank you.

**Comment by prio** [comment]
Hi. I needed this as well and have a tested version working on my own install. @tommoor shall I open a pull request?

**Reply by tommoor** [reply]
Hi yes, go for it.

**Reply by helmut72** [reply]
That would be great, thanks!

**Reply by not7cd** [reply]
@prio, I really look forward to it!

**Comment by not7cd** [comment]
You can selfhost grist. Would that mean that you need to add a field like this in Outline config? @tommoor 
![image](https://user-images.githubusercontent.com/4332632/206182743-cc9a5b82-b663-4569-838f-aa1bffccd9f2.png)

**Reply by tommoor** [reply]
Exactly, the idea is we'll add multiple services here.

**Comment by helmut72** [comment]
Feature is there, thank you!!
